kmcd-prelude.md

```diff
@@ -92,7 +67,12 @@ module KMCD-PRELUDE
 
          // "gold" collateral and joiner
          transact ADMIN Gem "gold" . init
-         transact ADMIN GemJoin "gold" . init
+         transact ADMIN GemJoin "gold" . constructor Vat Gem "gold"
+         transact ADMIN Gem     "gold" . initUser GemJoin "gold"
+         transact ADMIN Deploy . deployCollateral "gold" GemJoin "gold"
+
+         transact ADMIN Gem "gold" . initUser Vow
```

Comment:
I don't think either of these two lines are necessary? Only the `GemJoin` of "gold" needs a `Gem` balance, unless transferring "gold" to either the `Vow` or `Flap` is part of an attack (which it is not the case as far as I can recall)?

Reply:
Looks like they aren't. They're removed.